### PR TITLE
Transitioned Reverse a String in Scheme

### PR DESCRIPTION
**Pull request by jrg94**
Congrats on taking the first step to contributing to the [Sample Programs in Every Language][1] repository!

For simplicity, please make sure that your pull request includes one and only one sample program and language.

## Please Complete the Following

- [ ] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [ ] I named the pull request using `<Sample Program> in <Language>` format
- [ ] I created/updated the language README
  - [ ] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])
- [ ] I fixed #your-issue-number-here

## Notes

Feel free to put whatever you want here.

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

### CODE REVIEW

File: docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+title: Reverse a String in Scheme
+layout: default
+last-modified: 2018-12-24
+featured-image:
+tags: [scheme, reverse-a-string]
+authors:
+  - alexandra_woerner
+---
+
+In this article, we're tackling Reverse a String in Java!
```
Comment (awoerner92) [inline docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md]: Maybe a copy-paste error? ;) Should be Scheme
```suggestion
In this article, we're tackling Reverse a String in Scheme!
```

File: docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+The first thing you see there, is the display function. You’ll probably remember
+it from our Hello World experience, so have a look there, if you need a quick
+refresher of what it does.
+
+The argument to this function is the reverse-string function defined above. It
+receives the input from the command line, which is accessed with the function
+read-line, as its string argument. read-line reads a single line from the
+command line and converts it into a string, so it’s perfect for our application!
+
+If you replace (read-line) with a string, the program prints that string
```
Comment (awoerner92) [inline docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md]:
```suggestion
If you replace `(read-line)` with a string, the program prints that string
```

File: docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+### The Function Definition
+
+```scheme
+(define (reverse-string x)
+```
+
+The keyword define binds a function’s definition to the specified name. This
+keyword is followed by the name of the function and the function’s argument. In
+this particular case, the argument x is the string we want to reverse.[^1]
```
Comment (awoerner92) [inline docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md]:
```suggestion
this particular case, the argument *x* is the string we want to reverse.[^1]
```

File: docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+of the standard library. Finally, the reversed list is converted back to a string.
+This is also the return value of the function reverse-string.[^1]
+
+### The Display
+
+```scheme
+(display (reverse-string (read-line)))
+```
+
+The first thing you see there, is the display function. You’ll probably remember
```
Comment (awoerner92) [inline docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md]:
```suggestion
The first thing you see there, is the *display* function. You’ll probably remember
```

File: docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+it from our Hello World experience, so have a look there, if you need a quick
+refresher of what it does.
+
+The argument to this function is the reverse-string function defined above. It
```
Comment (awoerner92) [inline docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md]:
```suggestion
The argument to this function is the *reverse-string* function defined above. It
```

File: docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+receives the input from the command line, which is accessed with the function
+read-line, as its string argument. read-line reads a single line from the
```
Comment (awoerner92) [inline docs/projects/reverse-a-string/_posts/2018-12-24-scheme.md]:
```suggestion
*read-line*, as its string argument. *read-line* reads a single line from the
```

**Comment by jrg94** [comment]
All your comments look great!  We're in the process of duplicating all the old articles here (keeping  the old ones, of course). Then, people will be able to openly contribute to them here.

**Comment by jrg94** [comment]
Committed them all before realizing I could have batched them... Oooops.

**Pull request merged**